scripts/e2e.js

```diff
@@ -7,18 +7,15 @@ const getUpdatedPackageNames = require('./utils/getUpdatedPackageNames')
 getUpdatedPackageNames(packageNames => {
   const onlyCorePackages = packageNames.filter(name => name.startsWith('@tds/core-')).join(' ')
 
-  const { status } = spawnSync(
+  spawnSync(
     './node_modules/.bin/nightwatch',
     ['-c', './config/nightwatch.conf.js', '--env', 'headless'],
     {
       env: Object.assign({}, process.env, {
         PACKAGES: onlyCorePackages,
+        UPDATE_SCREENSHOTS: process.argv[2] === '-u',
       }),
       stdio: 'inherit',
     }
   )
-
-  if (status !== 0) {
-    process.exit(status)
-  }
```

Comment:
Please add this stuff with the `status` back. That was there on purpose for the `yarn prepr` task.

If the `spawnSync` for nightwatch here fails, we want the entire task to fail so that the `prepr` task stops, otherwise it ends up returning a 0 exit code, which is interpreted as a success.

Reply:
Ah, I see. Got it, I'll uncomment it.